apps/api/src/app.module.ts

```diff
@@ -100,13 +101,17 @@ const enterpriseModules = enterpriseImports();
 const modules = baseModules.concat(enterpriseModules);
 
 const providers: Provider[] = [
+  {
+    provide: APP_INTERCEPTOR,
+    useClass: ApiRateLimitInterceptor,
+  },
```

Comment:
The diff here is misleading - the `ApiRateLimitInterceptor` should be the first line of defense, so it's moved to be the first interceptor whilst  `ResourceThrottlerInterceptor` is added below